doc/releases/reef.rst

```diff
@@ -5,6 +5,499 @@ Reef
+  to accumulate extraneous "book-keeping" olh (object logical head) entries
+  and plain placeholder entries. In some specific scenarios where clients made
+  concurrent requests referencing the same object key, it was likely that a lot
+  of extra index entries would accumulate. When a significant number of these entries are
+  present in a single bucket index shard, they can cause high bucket listing
+  latencies and lifecycle processing failures. To check whether a versioned
+  bucket has unnecessary olh entries, users can now run ``radosgw-admin
+  bucket check olh``. If the ``--fix`` flag is used, the extra entries will
+  be safely removed. A distinct issue from the one described thus far, it is
+  also possible that some versioned buckets are maintaining extra unlinked
+  objects that are not listable from the S3/ Swift APIs. These extra objects
+  are typically a result of PUT requests that exited abnormally, in the middle
+  of a bucket index transaction - so the client would not have received a
+  successful response. Bugs in prior releases made these unlinked objects easy
+  to reproduce with any PUT request that was made on a bucket that was actively
+  resharding. Besides the extra space that these hidden, unlinked objects
+  consume, there can be another side effect in certain scenarios, caused by
+  the nature of the failure mode that produced them, where a client of a bucket
+  that was a victim of this bug may find the object associated with the key to
+  be in an inconsistent state. To check whether a versioned bucket has unlinked
+  entries, users can now run ``radosgw-admin bucket check unlinked``. If the
+  ``--fix`` flag is used, the unlinked objects will be safely removed. Finally,
+  a third issue made it possible for versioned bucket index stats to be
+  accounted inaccurately. The tooling for recalculating versioned bucket stats
+  also had a bug, and was not previously capable of fixing these inaccuracies.
+  This release resolves those issues and users can now expect that the existing
+  ``radosgw-admin bucket check`` command will produce correct results. We
+  recommend that users with versioned buckets, especially those that existed
+  on prior releases, use these new tools to check whether their buckets are
+  affected and to clean them up accordingly.
+
+* mgr/snap-schedule: For clusters with multiple CephFS file systems, all the
+  snap-schedule commands now expect the '--fs' argument.
+
+* RADOS: A POOL_APP_NOT_ENABLED health warning will now be reported if
+  the application is not enabled for the pool irrespective of whether
+  the pool is in use or not. Always tag a pool with an application
+  using ``ceph osd pool application enable`` command to avoid reporting
+  of POOL_APP_NOT_ENABLED health warning for that pool.
+  The user might temporarily mute this warning using
+  ``ceph health mute POOL_APP_NOT_ENABLED``.
+
+* An overview page for rgw to show the overall status of rgw components
```

Comment:
Should this and the next three entries be prefixed with "Dashboard: "?  It's not immediately clear what e.g.
> Added management support for RGW Multi-site and Ceph FS Subvolumes and groups

refers to.

Reply:
@idryomov want to get this PR merged so we don't have to wait forever for checks to pass, but I will fix this in a followup PR.

Reply:
@idryomov https://github.com/ceph/ceph/pull/54953

Reply:
Also s/rgw/RGW/

Reply:
@anthonyeleven I will address this in the followup PRs. Thx!
If you want to approve them, go ahead.

Reply:
Never mind, I see you already did. Thanks @anthonyeleven !